config/quickstarthub/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+resources:
+  - ../ingress-nginx
+  - ../metallb
+  - ../cert-manager
+  - ../external-dns
```

Comment:
```suggestion
```

Pretty sure external-dns is not used for anything.
Probably added during a spike for replacing the dns provider logic in mgc with external-dns

Reply:
This is definitely not required. I have removed it a couple of times, but seems to keep appearing back, probably bad through a rebase.

Reply:
we'll get rid of it for good here so. 🗡️